lte/gateway/docker/mme/Dockerfile.ubuntu18.04

```diff
@@ -52,61 +52,68 @@ RUN echo "Install 3rd party dependencies" && \
     apt-get install -y libsystemd-dev && \
     ln -s /usr/bin/python2.7 /usr/local/bin/python
 
-##### NETTLE
-RUN wget --quiet https://ftp.gnu.org/gnu/nettle/nettle-2.5.tar.gz && \
-    tar -xf nettle-2.5.tar.gz && \
-    cd nettle-2.5 && \
-    mkdir build && \
-    cd build/ && \
-    ../configure --disable-openssl --enable-shared --libdir=/usr/lib && \
-    make -j`nproc` && \
-    make install && \
-    ldconfig -v && \
-    cd / && \
-    wget --quiet https://www.gnupg.org/ftp/gcrypt/gnutls/v3.1/gnutls-3.1.23.tar.xz && \
-    tar xf gnutls-3.1.23.tar.xz && \
-    cd gnutls-3.1.23 && \
-    ./configure --with-libnettle-prefix=/usr --prefix=/usr && \
-    make -j`nproc` && \
-    make install && \
-    ldconfig -v
+RUN apt-get install -y python3-pip && \
+    pip3 install jinja2-cli
+
+# Add Converged MME sources to the container
+WORKDIR /patches
+COPY  lte/gateway/c/core/oai/patches/0001-opencoord.org.freeDiameter.patch .
```

Comment:
Nit: Since you replaced the dots below with absolute paths, would it be better to do the same for the patches?